main/streams/memory.c

```diff
@@ -489,9 +489,14 @@ static int php_stream_temp_cast(php_stream *stream, int castas, void **ret)
 		return FAILURE;
 	}
 
+	file = php_stream_fopen_tmpfile();
+	if (file == NULL) {
+		php_error_docref(NULL, E_WARNING, "Unable to create temporary file. Check permissions in temporary files directory.");
```

Comment:
> It can fail to various reasons, so maybe could simplify the msg or even omit it if php_stream_fopen_tmpfile() already throws a warning.

Message was simplified.